src/components/v5/common/ActionSidebar/partials/TransactionTable/TransactionTable.tsx

```diff
@@ -64,7 +72,7 @@ const TransactionTable: FC<TransactionTableProps> = ({ name }) => {
             fieldArrayMethods.append({
               amount: {
                 amount: '0',
-                tokenAddress: nativeToken?.tokenAddress || '',
+                tokenAddress: tokenAddressController.value || '',
```

Comment:
This line can be removed after implement this comment https://github.com/JoinColony/colonyCDapp/pull/1214/files#r1373098797